Номер3/Номер3/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Номер3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("enter x = ");
+            var x = double.Parse(Console.ReadLine());
+            Console.WriteLine(GoToCount(x));
+            Console.ReadKey();
+        }
+        static double GoToCount(double x)
```

Comment:
Вообще-то, _go to_ в программировании означает безусловную передачу управления. Лучше избегать таких названий. Уж лучше
```suggestion
        static double F(double x)
```
или так
```suggestion
        static double Calculate(double x)
```